test/continuum_absorption.jl

```diff
@@ -103,20 +103,21 @@ end
     @testset "Gray (2005) Fig 8.5$panel comparison" for panel in ["b", "c"]
         calculated, ref = Gray_opac_compare.Gray05_comparison_vals(panel,"Hminus_ff")
         @test all(calculated .≥ 0.0)
-        @test all(abs.(calculated - ref) .≤ Gray_opac_compare.Gray05_atols[panel])
+        @test assert_allclose(calculated, ref; rtol=0, atol=0.03)
     end
 end
 
 """
     _Hminus_bf_cross_section_Gray(λ)
```

Comment:
Since the Wishart data is already digitized, it might make slightly more sense to use it here than the Gray polynomials. But, I don't want to make more work for you (and it really doesn't matter at the end of the day)

Reply:
Yeah, this occurred to me but I decided it wasn't worth the effort.